### PR TITLE
dokan: Add version 1.3.1.1000

### PR DESCRIPTION
**Pull request by xeijin**
attempting again after issues with #3783

### CODE REVIEW

File: bucket/dokan.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            "64bit": {
+                "url": "https://github.com/dokan-dev/dokany/releases/download/v$version/Dokan_x64.msi"
+            },
+            "32bit": {
+                "url": "https://github.com/dokan-dev/dokany/releases/download/v$version/Dokan_x86.msi"
+            }
+        },
+        "extract_dir": "Dokan\\DokanLibrary-$matchHead"
+    },
+    "suggest": {
```
Comment (xeijin) [inline bucket/dokan.json]: @Ash258 why does your comment in old PR remove suggestion for vcredist2019? installation fails without it.

File: bucket/dokan.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+    "version": "1.3.1.1000",
+    "description": "User mode file system for Windows",
+    "homepage": "https://dokan-dev.github.io/",
+    "license": "LGPL-3.0-only",
+    "architecture": {
```
Comment (Ash258) [inline bucket/dokan.json]:
```suggestion
    "suggest": {
        "vcredist": "extras/vcredist2019"
    },
    "architecture": {
```

File: bucket/dokan.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            },
+            "32bit": {
+                "url": "https://github.com/dokan-dev/dokany/releases/download/v$version/Dokan_x86.msi"
+            }
+        },
+        "extract_dir": "Dokan\\DokanLibrary-$matchHead"
+    },
+    "suggest": {
+        "vcredist": "extras/vcredist2019"
+    }
```
Comment (Ash258) [inline bucket/dokan.json]:
```suggestion
    }
```

File: bucket/dokan.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            "hash": "1b89e2f3cf90902726a5d03a9694596a3a247d6908b74ab493469b259d8f8462"
+        },
+        "32bit": {
+            "url": "https://github.com/dokan-dev/dokany/releases/download/v1.3.1.1000/Dokan_x86.msi",
+            "hash": "b296e7bef4c6ca45f10b6c1b444ff2e38dec40c79753ca24f26f19c1a6aaaf0b"
+        }
+    },
+    "bin": [
+        "dokanctl.exe"
+    ],
```
Comment (Ash258) [inline bucket/dokan.json]:
```suggestion
    "bin": "dokanctl.exe",
```

File: bucket/dokan.json (hunk 3 above)
Comment (xeijin) [reply bucket/dokan.json]: Can you explain why you removed? other packages like `alacritty` have exactly the same?

Comment (Ash258) [reply bucket/dokan.json]: ~~re~~moved

Comment (xeijin) [reply bucket/dokan.json]: The installer does not work without vcredist2019 when testing on my local machine

Comment (xeijin) [reply bucket/dokan.json]: @Ash258 I don't understand ...

Comment (Ash258) [reply bucket/dokan.json]: I moved it above. Nothing was removed.

Comment (xeijin) [reply bucket/dokan.json]: @Ash258 ok thanks

**Comment by xeijin** [comment]
If you dont think it will work feel free to reject, hadn't realised that. Otherwise renamed, thanks.

**Pull request closed without merging**